docs/connect-a-frontend.md

```diff
@@ -0,0 +1,20 @@
+This demo will discuss the options we would have if we wanted to connect a frontend to the AEA. The following diagram illustrates 
```

Comment:
`the options we would have if we wanted to connect` > `the options we have to connect`